backend/registration-service/src/main/resources/application.properties

```diff
@@ -1,12 +1,16 @@
 spring.application.name=registration-service
 
-#  postgres db config
-spring.datasource.url=jdbc:postgresql://localhost:5432/unicorn_registration_db
-spring.datasource.username=
-spring.datasource.password=
-spring.jpa.hibernate.ddl-auto=create-drop
-spring.jpa.show-sql=true
+# enable liquibase
+spring.liquibase.change-log=classpath:db/changelog/db.changelog-master.xml
+
+# postgres db config
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
```

Comment:
This will be an environment specific value, so you'll want to look at moving this into a profile ideally.
Localhost will only work when running locally so maybe this should be a 'local' profile?